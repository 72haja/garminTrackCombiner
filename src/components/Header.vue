<script setup>
</script>

<template>
  <p class="h1 text-red">
    Garmin Track-Combiner
  </p>
</template>
<style lang='scss'>
</style>
