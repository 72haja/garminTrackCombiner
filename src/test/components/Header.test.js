import { describe } from 'vitest'
import { mount } from '@vue/test-utils'

import { createVuetify } from 'vuetify'

import Header from '../../components/Header.vue'

describe('mount App', () => {
  const vuetify = createVuetify()

  test('mount component', async () => {
    expect(Header).toBeTruthy()

    const wrapper = mount(Header, {
      global: {
        plugins: [vuetify]
      }
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
