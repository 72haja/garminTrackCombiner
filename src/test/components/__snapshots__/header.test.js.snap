// Vitest Snapshot v1

exports[`mount App > mount component 1`] = `"<p class=\\"text-h2 text-grey\\"> Garmin Track-Combiner </p>"`;
